### PR TITLE
Dst core/develop

### PR DESCRIPTION
**Pull request by JeremyFyke**
Hi @matprov - could I request a pull request into main, and then deployment of a new tag to climatedata.crim.ca?  I'd like to demo this version to a few parties in the next few days, if I can.

Note, I disabled pre-commit on merges to dst-core from individual sandbox branches.  Is there a way to just run pre-commit on merges to main?   I definitely value pre-commit but it's REALLY hard to manage pre-commit with new folks who are very new to software and don't understand git flow very well.  If you/I could just do pre-commit runs on less-frequent merges to main that would be super helpful on my end.  Your thoughts welcome.

Jeremy

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-    - id: black-jupyter
-      name: "Lint Jupyter notebooks using Black"
-      files: decision-support-tool.ipynb
-- repo: https://github.com/mwouts/jupytext
-  rev: v1.11.0
-  hooks:
-    - id: jupytext
-      name: "Generate a .py file from the notebook"
-      entry: jupytext --to py decision-support-tool.ipynb
-      pass_filenames: false
```
Comment (matprov) [inline .pre-commit-config.yaml:22]: Please keep this file. If you absolutely need to deactivate, simply uninstall `pre-commit` from your local machine. This validation might be sometimes annoying for small changes/wip, but before deploying it's really important to make sure nothing broke and we don't have regression. I would say that you can bypass this system for `feat-branch -> develop`, but for `develop -> master` I prefer to keep this mecanism.

**Review by matprov — CHANGES_REQUESTED** [review]
@JeremyFyke Two little things to change but overall good PR. Once you'll have made the modification, I will deploy the new version.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (JeremyFyke) [reply .pre-commit-config.yaml:22]: Thanks for these.  Will revert these changes, and work on workflow on my end as per your comments.

File: .gitignore [hunk 2]
```diff
@@ -5,4 +5,5 @@ __pycache__/textart.cpython-37.pyc
 __pycache__/textart.cpython-37.pyc
 *.pyc
 .ipynb_checkpoints/*
-.DS_Store
\ No newline at end of file
+.DS_Store
+*.py
```
[the remark this answers is not in the document]
Comment (JeremyFyke) [reply .gitignore:9]: ok

**Review by matprov — APPROVED** [review]

**Comment by matprov** [comment]
Hi @JeremyFyke The new version is deployed. I would suggest changing the image of the new hazards panel. It's a bit confusing for users to always see new image for panels but this image is used twice.

**Comment by JeremyFyke** [comment]
This is great, thanks.  One thing I noticed, is that on final summary stage, the accordion-style expandable list of resources expands below the range of the page.  Seeing these is important (in fact, this is the primary outcome of the tool).  Could we make this web page adaptively sized, so the height of the page grows as needed?  Or alternatively, for the moment, just expand the static height of the html container so it definitely captures reasonable expansions of this accordion tool?  A long term solution could be to make this accordion tool a separate tab entirely, so it doesn't 'stack' below the SanKey diagram.

**Comment by matprov** [comment]
Hi @JeremyFyke Yes, the layout will be fixed in order to show all the content from the accordion-style view. It will be based on the content, so the page will grow when accordion is expanded. 

Deployments takes couple of minutes. It's only a matter of building locally the docker image to make sure the app itself works, then pushing changes in staging. I prefer to have multiple smaller deploys than waiting for bigger ones with bigger changes.

**Comment by JeremyFyke** [comment]
> Hi @JeremyFyke Yes, the layout will be fixed in order to show all the content from the accordion-style view. It will be based on the content, so the page will grow when accordion is expanded.
> 
> Deployments takes couple of minutes. It's only a matter of building locally the docker image to make sure the app itself works, then pushing changes in staging. I prefer to have multiple smaller deploys than waiting for bigger ones with bigger changes.

Hi @matprov OK thanks.   This is great, can you let me know when the 'page growing' option is enabled?  This should be last thing I need before I show this tool to some important building sector experts for review.  Related: I would also be happier with multiple smaller deploys as well (so feedback from these reviews is fairly rapidly available in the updated development site app).

**Pull request merged**